applications: Added support for groups in the Matter bridge (#13084)
nrfconnect/sdk-nrf: doc/nrf/releases_and_maturity/known_issues.rst

The issues in this section are related to the :ref:`ug_matter` protocol.

.. rst-class:: v2-5-0 v2-4-2 v2-4-1 v2-4-0 v2-3-0 v2-2-0 v2-1-4 v2-1-3 v2-1-2

> KRKNWK-17360: The groupcast communication does not work for multiple endpoints being part of a group on a single Matter node

Reviewer: ```suggestion
KRKNWK-17360: Groupcast communication does not work for multiple endpoints thare are part of a group on a single Matter node
```

>   The logic in a Matter core implementation handles commands status in a wrong way for those targeted to a group.

Reviewer: ```suggestion
  The Matter core implementation handles commands status in a wrong way for those targeted to a group.
```

>   The issue is only visible in case of adding multiple endpoints, existing on the same Matter node, to the one group.
>   It results in application crash after receiving a group command.

Reviewer: ```suggestion
  This issue is only visible when adding multiple endpoints that exist on the same Matter node to the same group, and results in an application crash after receiving a group command.
```

>   In case of adding multiple Matter nodes with a single endpoint each to the group, the communication works correctly.

Reviewer: ```suggestion
  When adding multiple Matter nodes with a single endpoint each to the same group, the communication works correctly.
```

>   **Workaround:** Manually cherry-pick and apply commit from the main branch (commit hash: ``TODO: fill after merging sdk-connectedhomeip PR``).

Reviewer: ```suggestion
  **Workaround:** Manually cherry-pick and apply the commit with the fix to ``sdk-connectedhomeip`` (commit hash: ``TODO: fill after merging sdk-connectedhomeip PR``).
```
